src/domains/job/recruitment/components/form/WriteRecruitmentPostingForm.jsx

```
@@ -56,6 +58,14 @@ const WriteRecruitmentPostingForm = ({ onSubmit, defaultValues }) => {
     onSubmit(formData)
   }
 
+  const onSave = () => {
+    const formData = {
+      nickname: control._formValues.nickname || '',
+      title: control._formValues.title || '',
+      text: control._formValues.text,
+    }
+    handleSaveDraft(formData, onSaveDraft, 'recruitment')
```

ESLint

[error] 'onSaveDraft' is not defined.

(no-undef)